rust/relay/src/server.rs

```diff
@@ -774,11 +810,12 @@ where
         self.data_relayed_counter.add(data.len() as u64, &[]);
         self.data_relayed += data.len() as u64;
 
-        self.pending_commands.push_back(Command::ForwardData {
-            id: channel.allocation,
-            data: data.to_vec(),
-            receiver: channel.peer_address,
-        });
+        self.pending_commands
+            .push_back(Command::ForwardDataClientToPeer {
```

Comment:
Is this just a `Deque`? What was the reason not to limit the capacity?

Reply:
It is emptied on every iteration of `Eventloop::poll` and only a small number of `Command`s can build up.

Reply:
How small? 100?

Reply:
More like 2-3.

Reply:
Could we panic if it hits 100?